community/contributing/back-end-code.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: single
+title:  "Code Conventions for Server-side Components"
+sidebar:
+  nav: community
+redirect_from: /docs/back-end-code
```

Comment:
Is this redirect necessary?

Reply:
No idea. I was just going from other docs

Reply:
Ah we use it when we move docs around to keep old links working. I'd probably drop it for now.

Reply:
Agreed that it's not needed.